packages/protoplugin-test/src/transpile.test.ts

```diff
@@ -185,4 +185,21 @@ describe("transpile", function () {
       "export declare function foo(): Foo;",
     ]);
   });
+
+  test("print with tagged template literal works correctly", () => {
```

Comment:
Can you move this test to `generated-file.test.ts` also?

Reply:
@smaye81 Oops it's duplicated with [this test](https://github.com/yukukotani/protobuf-es/blob/773d47585173870442936e08861757875dffc489/packages/protoplugin-test/src/generated-file.test.ts#L45:L45). I've removed it.